onnxruntime/core/providers/nv_tensorrt_rtx/nv_provider_factory.cc

```diff
@@ -100,6 +99,35 @@ std::unique_ptr<IExecutionProvider> NvProviderFactory::CreateProvider() {
   return std::make_unique<NvExecutionProvider>(info_);
 }
 
+std::unique_ptr<IExecutionProvider> NvProviderFactory::CreateProvider(const OrtSessionOptions& session_options, const OrtLogger& session_logger) {
```

Comment:
Is this function ever used?

Reply:
Ok, it seems this is to address the review comment about future use with Compile API. makes sense.